validator/client/beacon-api/sync_committee.go

```diff
@@ -88,8 +96,12 @@ func (c *beaconApiValidatorClient) getSyncCommitteeContribution(
 	url := buildURL("/eth/v1/validator/sync_committee_contribution", params)
 
 	var resp apimiddleware.ProduceSyncCommitteeContributionResponseJson
-	if _, err := c.jsonRestHandler.GetRestJsonResponse(ctx, url, &resp); err != nil {
-		return nil, errors.Wrap(err, "failed to query GET REST endpoint")
+	errJson, err := c.jsonRestHandler.Get(ctx, url, &resp)
+	if err != nil {
+		return nil, errors.Wrapf(err, msgUnexpectedError)
+	}
+	if errJson != nil {
+		return nil, errJson
 	}
```

Comment:
Same here:
```suggestion
	if err != nil {
		return nil, errors.Wrap(err, msgUnexpectedError)
	}
	if errJson != nil {
		return nil, errors.Wrap(errJson, msgRequestFailed)
	}
```

Reply:
`Wrapf` is a mistake, but `errJson` doesn't have to be wrapped as it contains `HTTP request unsuccessful` as part of its error message